recipes/xkbcommon/all/conanfile.py

```diff
@@ -39,7 +40,7 @@ class XkbcommonConan(ConanFile):
 
     @property
     def _has_build_profile(self):
-        return hasattr(self, "settings_build")
+        return hasattr(self, "settings_build") and cross_building(self)
```

Comment:
No change is required here I would say.

Reply:
You're right, that would change the method meaning. I was trying to fix the issue that you also ran into in #20542. I'm not sure id didn't cause other problems. I've reverted.